{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "important"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "SUSE CVE-2020-28975",
				"title": "Title"
			},
			{
				"category": "description",
				"text": "** DISPUTED ** svm_predict_values in svm.cpp in Libsvm v324, as used in scikit-learn 0.23.2 and other products, allows attackers to cause a denial of service (segmentation fault) via a crafted model SVM (introduced via pickle, json, or any other model permanence standard) with a large value in the _n_support array. NOTE: the scikit-learn vendor's position is that the behavior can only occur if the library's API is violated by an application that changes a private attribute.",
				"title": "Description of the CVE"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "CVE-2020-28975",
				"url": "https://www.suse.com/security/cve/CVE-2020-28975"
			},
			{
				"category": "external",
				"summary": "SUSE Security Ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "external",
				"summary": "SUSE Bug 1179081 for CVE-2020-28975",
				"url": "https://bugzilla.suse.com/1179081"
			}
		],
		"title": "SUSE CVE CVE-2020-28975",
		"tracking": {
			"current_release_date": "2023-02-15T03:51:52Z",
			"generator": {
				"date": "2023-02-15T03:51:52Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf-vex.pl",
					"version": "1"
				}
			},
			"id": "CVE-2020-28975",
			"initial_release_date": "2023-02-15T03:51:52Z",
			"revision_history": [
				{
					"date": "2023-02-15T03:51:52Z",
					"number": "2",
					"summary": "Current version"
				}
			],
			"status": "interim",
			"version": "2"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "openSUSE Tumbleweed",
								"product": {
									"name": "openSUSE Tumbleweed",
									"product_id": "openSUSE Tumbleweed",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:tumbleweed"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python310-scikit-learn-1.0.2-1.1",
								"product": {
									"name": "python310-scikit-learn-1.0.2-1.1",
									"product_id": "python310-scikit-learn-1.0.2-1.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:scikit-learn:scikit-learn:1.0.2:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python38-scikit-learn-1.0.2-1.1",
								"product": {
									"name": "python38-scikit-learn-1.0.2-1.1",
									"product_id": "python38-scikit-learn-1.0.2-1.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:scikit-learn:scikit-learn:1.0.2:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python39-scikit-learn-1.0.2-1.1",
								"product": {
									"name": "python39-scikit-learn-1.0.2-1.1",
									"product_id": "python39-scikit-learn-1.0.2-1.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:scikit-learn:scikit-learn:1.0.2:*:*:*:*:*:*:*"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-scikit-learn-1.0.2-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-scikit-learn-1.0.2-1.1"
				},
				"product_reference": "python310-scikit-learn-1.0.2-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python38-scikit-learn-1.0.2-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python38-scikit-learn-1.0.2-1.1"
				},
				"product_reference": "python38-scikit-learn-1.0.2-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python39-scikit-learn-1.0.2-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python39-scikit-learn-1.0.2-1.1"
				},
				"product_reference": "python39-scikit-learn-1.0.2-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2020-28975",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-28975"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "** DISPUTED ** svm_predict_values in svm.cpp in Libsvm v324, as used in scikit-learn 0.23.2 and other products, allows attackers to cause a denial of service (segmentation fault) via a crafted model SVM (introduced via pickle, json, or any other model permanence standard) with a large value in the _n_support array. NOTE: the scikit-learn vendor's position is that the behavior can only occur if the library's API is violated by an application that changes a private attribute.",
					"title": "Vulnerability description"
				}
			],
			"product_status": {
				"fixed": [
					"openSUSE Tumbleweed:python310-scikit-learn-1.0.2-1.1",
					"openSUSE Tumbleweed:python38-scikit-learn-1.0.2-1.1",
					"openSUSE Tumbleweed:python39-scikit-learn-1.0.2-1.1"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-28975",
					"url": "https://www.suse.com/security/cve/CVE-2020-28975"
				},
				{
					"category": "external",
					"summary": "SUSE Security Ratings",
					"url": "https://www.suse.com/support/security/rating/"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1179081 for CVE-2020-28975",
					"url": "https://bugzilla.suse.com/1179081"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-scikit-learn-1.0.2-1.1",
						"openSUSE Tumbleweed:python38-scikit-learn-1.0.2-1.1",
						"openSUSE Tumbleweed:python39-scikit-learn-1.0.2-1.1"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:python310-scikit-learn-1.0.2-1.1",
						"openSUSE Tumbleweed:python38-scikit-learn-1.0.2-1.1",
						"openSUSE Tumbleweed:python39-scikit-learn-1.0.2-1.1"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2020-11-23T07:16:34Z",
					"details": "important"
				}
			],
			"title": "CVE-2020-28975"
		}
	]
}
